Aspire Budgeting/Models/File.swift

```diff
@@ -7,14 +7,21 @@
 //
 
 import Foundation
+import GoogleAPIClientForREST
 
 class File: Identifiable {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mohitathwani/aspirebudgeting/blob/c60f90ed59460d52ef9c97dbb14a1cc7a2b431bb/Aspire Budgeting/Models/File.swift#L12"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>